Gems/ScriptAutomation/Code/Source/ScriptAutomationSystemComponent.cpp

```diff
@@ -177,12 +177,13 @@ namespace AZ::ScriptAutomation
 
     void ScriptAutomationSystemComponent::SetIdleFrames(int numFrames)
     {
-        AZ_Assert(m_scriptIdleSeconds == 0, "m_scriptIdleFrames is being stomped");
+        AZ_Assert(m_scriptIdleFrames <= 0, "m_scriptIdleFrames is being stomped");
```

Comment:
An interesting assert description about m_scriptIdleFrames being stomped. Could it just not be bad code somewhere?

Reply:
It's not currently being stomped, this is just to catch that happening in the future.